NExtends/Html/HtmlUtility.cs

```diff
@@ -0,0 +1,213 @@
+            { "span", new string[]       { "style", "class" } },
+            { "br", new string[]         { "style", "class" } },
+            { "hr", new string[]         { "style", "class" } },
+            { "label", new string[]      { "style", "class" } },
+
+            { "h1", new string[]         { "style", "class" } },
+            { "h2", new string[]         { "style", "class" } },
+            { "h3", new string[]         { "style", "class" } },
+            { "h4", new string[]         { "style", "class" } },
+            { "h5", new string[]         { "style", "class" } },
+            { "h6", new string[]         { "style", "class" } },
+
+            { "font", new string[]       { "style", "class", "color", "face", "size" } },
+            { "strong", new string[]     { "style", "class" } },
+            { "b", new string[]          { "style", "class" } },
+            { "em", new string[]         { "style", "class" } },
+            { "i", new string[]          { "style", "class" } },
+            { "u", new string[]          { "style", "class" } },
+            { "strike", new string[]     { "style", "class" } },
+            { "ol", new string[]         { "style", "class" } },
+            { "ul", new string[]         { "style", "class" } },
+            { "li", new string[]         { "style", "class" } },
+            { "blockquote", new string[] { "style", "class" } },
+            { "code", new string[]       { "style", "class" } },
+            { "pre", new string[]       { "style", "class" } },
+
+            { "a", new string[]          { "style", "class", "href", "title" } },
+            { "img", new string[]        { "style", "class", "src", "height", "width", "alt", "title", "hspace", "vspace", "border" } },
+
+            { "table", new string[]      { "style", "class" } },
+            { "thead", new string[]      { "style", "class" } },
+            { "tbody", new string[]      { "style", "class" } },
+            { "tfoot", new string[]      { "style", "class" } },
+            { "th", new string[]         { "style", "class", "scope" } },
+            { "tr", new string[]         { "style", "class" } },
+            { "td", new string[]         { "style", "class", "colspan" } },
+
+            { "q", new string[]          { "style", "class", "cite" } },
+            { "cite", new string[]       { "style", "class" } },
+            { "abbr", new string[]       { "style", "class" } },
+            { "acronym", new string[]    { "style", "class" } },
+            { "del", new string[]        { "style", "class" } },
+            { "ins", new string[]        { "style", "class" } }
+        };
+
+        /// <summary>
+        /// Takes raw HTML input and cleans against a whitelist
+        /// </summary>
+        /// <param name="source">Html source</param>
+        /// <returns>Clean output</returns>
+        public string SanitizeHtml(string source)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 34 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3776)